crypto/ts/ts_verify_ctx.c

```diff
@@ -70,6 +70,8 @@ STACK_OF(X509) *TS_VERIFY_CTX_set_certs(TS_VERIFY_CTX *ctx,
 unsigned char *TS_VERIFY_CTX_set_imprint(TS_VERIFY_CTX *ctx,
                                          unsigned char *hexstr, long len)
 {
+    if (ctx->imprint)
+        OPENSSL_free(ctx->imprint);
```

Comment:
This should just be:
```c
    OPENSSL_free(ctx->imprint);
```

Our free calls handle NULLs properly.

We shouldn't need a separate pull request for 1.1.1.

Reply:
> We shouldn't need a separate pull request for 1.1.1.

Should I close the PR for 1.1.1?